packit_service/worker/helpers/build/copr_build.py

```diff
@@ -212,6 +212,14 @@ def tests_targets_all_mapped(self) -> Set[str]:
             targets.update(self.build_target2test_targets(chroot))
         return targets
 
+    @property
+    def copr_project_path(self):
+        return f"{self.job_owner}/{self.job_project}"
+
+    @property
+    def forge_project_path(self):
+        return f"{self.project.service.hostname}/{self.project.namespace}/{self.project.repo}"
```

Comment:
nice improvement, I am only thinking about the naming, I would maybe go with `configured_copr_project` and `forge_project` without the `path` suffix, but naming is hard, so I am not sure :D